### PR TITLE
Implement carousel and overwrite in Download()

### PR DESCRIPTION
**Pull request by fronbasal**
This PR adds an overwrite switch and a carousel download function. I did not transfer the code into methods because of the amount of variables that are required by the code itself. If you have any recommendations or tipps to enhance this PR I would be happy to implement them.

### CODE REVIEW

File: media.go [hunk 1]
```diff
@@ -137,6 +137,16 @@ func getname(name string) string {
 	return name
 }
 
+func getnameoverwrite(name string) string {
```
Comment (ahmdrz) [inline media.go]: It must be more readable.
like `getNameOverwrite`.

Also I think it's better to change name of the file and add a simple number like unix time.

Comment (fronbasal) [reply media.go]: Sounds good. I will take care of this as soon as I get some time.

**Comment by ahmdrz** [comment]
Thank you @fronbasal
Welcome to goinsta contributers.

**Comment by fronbasal** [comment]
Thank you!

**Pull request merged**